src/Microsoft.SqlTools.ServiceLayer/Utility/ServiceLayerCommandOptions.cs

```diff
@@ -0,0 +1,34 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Globalization;
+using Microsoft.SqlTools.Hosting.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.Utility
+{
+    class ServiceLayerCommandOptions : CommandOptions
+    {
+        private static string ServiceLayerServiceName = "MicrosoftSqlToolsServiceLayer.exe";
```

Comment:
Is this class really needed? Can't you just change where the SqlToolsServiceLayer code uses CommandOptions and pass in the exe name?

Reply:
I added this class mainly because within SetLocale of this class, we set an internal culture (Sr.Culture = CultureInfo.CurrentCulture) specifically used in service layer tests. I don't need to / am not able to when handling the locale for command option usage in credentials and serialization since SR is internal to the service layer and is used in service layer specific tests.